packages/11ty/_plugins/figures/tests/__fixtures__/figures/zoom/manifest.json

```diff
@@ -0,0 +1,58 @@
+{
+  "@context": "http://iiif.io/api/presentation/3/context.json",
+  "id": "http://localhost:8080/iiif/zoom/manifest.json",
+  "type": "Manifest",
+  "behavior": [
+    [
+      "continuous",
+      "sequence"
+    ]
+  ],
```

Comment:
The `behavior` property is only valid on `sequences`, no?

Reply:
true!